RHEL/6/input/remediations/bash/package_samba_removed.sh

```diff
@@ -0,0 +1,4 @@
+# platform = Red Hat Enterprise Linux 6
+if rpm -qa | grep -q samba; then
+	yum -y remove samba
+fi
```

Comment:
Please update this remediation script to look like the following:
https://github.com/OpenSCAP/scap-security-guide/blob/master/RHEL/6/input/remediations/bash/package_ypserv_removed.sh